assets/js/components/optin.js

```diff
@@ -57,15 +57,15 @@ class Optin extends Component {
 			method: 'POST',
 		} )
 			.then( () => {
-				if ( !! checked && ! this.state.scriptOnPage ) {
-					const { document } = window;
+				const { document } = window;
 
-					if ( ! document ) {
-						return;
-					}
+				if ( ! document ) {
```

Comment:
Could this `document` check not remain inside the `if` like it was before? It's only used inside of that clause.

Reply:
If document isn't available I figured it was best to return early, but I can move it back in I suppose 😄

Reply:
Change made 😄